api/mapping/views.py

```diff
@@ -279,6 +283,7 @@ class DatasetFilterView(generics.ListAPIView):
     filter_backends = [DjangoFilterBackend]
     filterset_fields = {
         "id": ["in"],
+        "data_partner": ["in","exact"],
```

Comment:
Let's not do this change. `DatasetFilterView` gets merged into `DatasetListView` by #392. 
```suggestion
```